docs/schema/concepts/scoping-rules.mdx

```diff
@@ -14,32 +13,20 @@ import ExampleSubjectiveHeadingScoping from '!!raw-loader!@site/docs/_examples/t
 import ExampleSubjectiveUsagePurposeScoping from '!!raw-loader!@site/docs/_examples/transportation/docusaurus/subjective-usage-purpose-scoping.yaml';
 import ExampleSubjectiveStatusScoping from '!!raw-loader!@site/docs/_examples/transportation/docusaurus/subjective-status-scoping.yaml';
 import ExampleSubjectiveVehicleAttributesScoping from '!!raw-loader!@site/docs/_examples/transportation/docusaurus/subjective-vehicle-attributes-scoping.yaml';
-import ExampleLanesAbsoluteForm from '!!raw-loader!@site/docs/_examples/transportation/docusaurus/lanes-absolute-form.yaml';
 
-## Scoped and rule-based properties
 
-In the real world, many facts and rules affecting entities have
-only a partial application, meaning they don't apply everywhere, or
-they don't apply at all times, or to everyone, or to all sets of
-external conditions. For example, access restrictions on a road segment
-might not apply to all people or all kinds of vehicles, or they might
-vary according to the day of the week. <!-- add another example that is not transportation focused -->
+In the real world, many facts and rules affecting entities have only a partial application, meaning they don't apply everywhere, or they don't apply at all times, or to everyone, or to all sets of external conditions. For example, access restrictions on a road segment might not apply to all people or all kinds of vehicles, or they might vary according to the day of the week.
+<!-- add another example that is not transportation focused -->
 
 The Overture schema uses two related concepts to capture the partial application of facts and rules: scoped values and rule-based properties.
 
-### Scoped values and scoping properties
+## Scoped values and scoping properties
 
-A *scoped* value is a value which only applies within a limited scope.
-Most scoped values are rules in the rule lists of
-[rule-based properties](#rules-and-rule-based-properties). However,
-scoped values also exist outside of rule-based properties. For
-example, a signpost property belonging to a road segment might be
-geometrically scoped to its position along the road.
+A *scoped* value is a value which only applies within a limited scope. Most scoped values are rules in the rule lists of [rule-based properties](#rules-and-rule-based-properties). However, scoped values also exist outside of rule-based properties. For example, a signpost property belonging to a road segment might be geometrically scoped to its position along the road.
```

Comment:
Now that we have concretized the abstract idea of "signpost" into something final (destinations), can we update the "For example, a signpost ..." text to reference destinations? (And possibly change the "might be" to a a stronger statement like "must be" assuming the schema agrees with that.)